extensions/amazon-lambda/deployment/src/main/java/io/quarkus/amazon/lambda/deployment/LambdaConfig.java

```diff
@@ -1,12 +1,12 @@
-package io.quarkus.amazon.lambda.runtime;
+package io.quarkus.amazon.lambda.deployment;
 
 import java.util.Optional;
 
 import io.quarkus.runtime.annotations.ConfigItem;
 import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
 
-@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+@ConfigRoot(phase = ConfigPhase.BUILD_TIME)
```

Comment:
This should really be runtime. The idea with the runtime config of the handler name was that you could create a single native image with multiple handlers, and use environment variables to select the handler to use at deployment time.

If this is build time and you have a deployment that has 10 different functions you would need 10 different native image builds.

Reply:
You shouldn't have a deployment with 10 different functions.

Reply:
Moving this back to runtime would also require moving error condition checks from build time to runtime which can be very problematic.  You don't want to have to debug build time issues at runtime.